packages/adapter-netlify/index.js

```diff
@@ -32,7 +31,7 @@ export default function (options) {
 			/** @type {BuildOptions} */
 			const defaultOptions = {
 				entryPoints: ['.svelte-kit/netlify/entry.js'],
-				outfile: join(functions, 'render/index.js'),
+				outfile: '.netlify/functions-internal/__render.js',
```

Comment:
Is there some documentation on the Netlify site that talks about the `.netlify` directory? What would go in it, how it's used, etc.?

Reply:
I'm off this week so can't check, but I don't _think_ the internal functions directory is documented yet. It was only just added a few weeks ago. Generally the `.netlify` directory is for internal use and is used for compiled functions and caches. The new internal functions directory is meant for auto-generated  functions, mainly those created by Netlify build plugins

Reply:
Is `functions-internal` something we just made up or some directory the Netlify CLI expects and uses in some way? This could use a code comment explaining what this directory is

Reply:
It's a special directory that the Netlify CLI looks in for extra functions. It is not yet documented. It's meant to be used for auto-generated functions like these which are expected to be `.gitignore`d. It's normally used by build plugins, but that's effectively what this is. See https://github.com/netlify/build/pull/3213 I'll add a comment to this effect.